src/v4/hook/useFile.ts

```diff
@@ -10,13 +11,12 @@ interface useFileProps {
 export const useFile = () => {
   const getImage = useCallback(
     async ({ fileId, imageSize = ImageSizeState.medium }: useFileProps) => {
-      if (!fileId) return undefined;
+      if (!fileId) return defaultAvatarUri;
       const file = await FileRepository.getFile(fileId);
-      if (!file) return undefined;
-      const newImageUrl = FileRepository.fileUrlWithSize(
-        file.data.fileUrl,
-        imageSize
-      );
+      if (!file) return defaultAvatarUri;
+      const newImageUrl =
+        FileRepository.fileUrlWithSize(file.data.fileUrl, imageSize) ??
+        defaultAvatarUri;
       return newImageUrl;
```

Comment:
Why not just return directly ?

```suggestion
      return FileRepository.fileUrlWithSize(file.data.fileUrl, imageSize) ?? defaultAvatarUri;
```

Reply:
There is no special reason xD. 
May be my habbit xD. 
I will change this later.